docs/help-topics/stonesoup-import/stonesoup-import.yml

```diff
@@ -30,32 +29,51 @@
     * Spring Boot
 
     ***
+
     **Using a code sample**
-    1. To start with a code sample, click **Start with a sample**, then go ahead and explore or test various languages and frameworks.         
-    2. Click any of the options on the **Select a sample** page, then follow the resulting instructions.
+    To start with a code sample, in the code sample **Filter by keyword** field, enter the name of the code you want to work with, or click **Import sample** on any of the options displayed. Follow the resulting instructions to explore or test whatever language and framework you choose.
 
 
 - name: stonesoup-import-configure-component
   tags:
   title: Configure components
   content: |-
-    We analyze your code, then define and apply deployment settings for each of your application components. Remember, a “component” is an image that we build from source code in a repository. An application is a collection of these components working together.
+    We analyze your code, then define and apply deployment settings for each of your application components. Remember, a “component” is a container image that we build from source code in a repository. An application is a collection of these components working together.
+
+    Before you create and deploy your application, look over the values and options that display here. You can rename any component from this view, too.
+
+    - Under **Application details**, check **Name**. This is the application name we created for you, but if you want to change it, you can do so in this field.
+
+    In the **Components** section, look over these items:
+
+    - **Component name**: Is it correct? Click the GitHub link under this field if you want to check your repository URL.
+    - **Runtime**: Did we detect the right one? If so, you should notice that we suggest appropriate deployment options. If you want to change your runtime, expand the menu next to your runtime name and make a selection.
 
-    Before you create and deploy your application, look over the values that display here. You can rename any component from this view, too. In particular, pay attention to the following fields:
+    Under **Build & deploy configuration**, review these settings:
 
     - **Target port**: Is it correct? If not, click in the port field to modify it.
-    - **CPUs**, **memory**, and **instances**: Choose how many of each of these you want for your application, depending on your subscription.
+    - **Dockerfile**: If you specified **Dockerfile** as your runtime, make sure we detected the right one. Click in the field if you need to modify it. Don't worry! We hide this field if you chose a different runtime.
```

Comment:
I don't think we need the "Don't worry!" here.  Maybe just "If you chose a different runtime, we hide this field." @jseseCCS